.dscanner.ini

```diff
@@ -207,11 +207,12 @@ assert_without_msg="-etc.c.SQL_,\
 -std.uuid,\
 -std.variant,\
 -std.windows.registry,\
--std.xml"
+-std.xml,\
+-std.sumtype"
```

Comment:
I'd prefer to have std.sumtype in the correct place (after std.string) to keep the alphabetical order. This applies to all other changes in this file too.

Reply:
Good catch. Fixed.